querygen/querygen.go

```diff
@@ -34,7 +34,8 @@ func GenBuildPipelineRunCreatedQuery(index string) string {
 				`"responseObject.metadata.resourceVersion"="*"`,
 		).
 		WithEventExpr(`"Build PipelineRun created"`).
-		WithFields("namespace", "application", "component", "repo", "commit_sha", "target_branch").
+		WithFields("namespace", "application", "component", "repo", "commit_sha", "target_branch",
+			"git_trigger_event_type", "git_trigger_provider").
```

Comment:
I'm thinking we should add these fields to the other build PipelineRun related queries as well. They might be useful somehow.

Reply:
Now it's in all of the Build piplineRun's related queries,
I also added pipeline log-url, other than that i'm not sure what is there more to add (there's already the commit sha, branch, repo, target branch, etc)